tools/ReleaseEngineering/ServicingPipeline.ps1

```diff
@@ -61,6 +61,17 @@ Function Reject() {
     $Host.ExitNestedPrompt()
 }
 
+If ([String]::IsNullOrEmpty($Version)) {
+    $BranchVersionRegex = [Regex]"^release-(\d+(\.\d+)+)$"
+    $Branch = & git rev-parse --abbrev-ref HEAD
+    $Version = $BranchVersionRegex.Match($Branch).Groups[1].Value
+    If ([String]::IsNullOrEmpty($Version)) {

```

Comment:
You could abbreviate that as:
```pwsh
if ($Branch -notmatch "^release-(\d+(\.\d+)+)$") {
```
and use `$Matches.1`.

Reply:
Thanks! I stuck with the prevailing style for matches in this file -- in general, I do not like magic implicit variables like `$Match`. It's also my least favorite part of bash.